static/app/views/dashboardsV2/datasetConfig/errorsAndTransactions.tsx

```diff
@@ -10,6 +11,12 @@ export const ErrorsAndTransactionsConfig: DatasetConfig<
   EventsStats | MultiSeriesEventsStats,
   TableData | EventsTableData
 > = {
+  getCustomFieldRenderer: (field, meta, contextualProps) => {
```

Comment:
I just remembered this [ticket](https://getsentry.atlassian.net/browse/DD-753?atlOrigin=eyJpIjoiNzYxMzAzNWRiOGQ5NGNjYTg1M2JkNTYxYjkxMjY4ZjMiLCJwIjoiaiJ9), do you think we can pull this change in here?

Reply:
Pulled that stuff out into the config 👍